grow/commands/convert.py

```diff
@@ -0,0 +1,20 @@
+from grow.pods import pods
+from grow.pods import storage
+from grow.conversion import *
+import click
+import os
+
+
+@click.command()
+@click.argument('pod_path', default='.')
+@click.option('--version', help='What version to convert files to.')
+def convert(pod_path, version):
+    """Converts pod files from an earlier version of Grow."""
+    root = os.path.abspath(os.path.join(os.getcwd(), pod_path))
+    pod = pods.Pod(root, storage=storage.FileStorage)
+
+    if version == '0.1.0':
```

Comment:
I wonder if we should be more declarative about this command and use names for the conversion methods rather than versions. For example, if a newer version of Grow detects a known-kind of parse error with YAML due to the removal of the split document feature, we could advise people to run `grow convert split-documents` (or some better name) rather than the version? That might be more clear for people?